clients/fides-js/src/fides.ts

```diff
@@ -71,28 +85,45 @@ declare global {
 // eslint-disable-next-line no-underscore-dangle,@typescript-eslint/naming-convention
 let _Fides: Fides;
 
-// Initialize the global Fides object with the given configuration values
-const init = (config: FidesConfig) => {
-  // Configure the default values
+/**
+ * Initialize the global Fides object with the given configuration values
+ */
+const init = async (config: FidesConfig) => {
+  // Configure the default consent values
   const context = getConsentContext();
-  const defaults = makeConsentDefaults({
+  const consentDefaults = makeConsentDefaults({
     config: config.consent,
     context,
   });
 
   // Load any existing user preferences from the browser cookie
-  const cookie = getOrMakeFidesCookie(defaults);
+  const cookie = getOrMakeFidesCookie(consentDefaults);
+
+  await initFidesConsent(config);
```

Comment:
nit: How about `initOverlay`?

Reply:
wow, I'm only just realizing that we may have been interpreting `overlay` differently this whole time! I think your definition is what `fides-js` overlays onto a customer's website, while I had always been thinking of it as specifically the modal (because there are ModalOverlays). a lot of things are suddenly making more sense!